README.md

```diff
@@ -71,6 +71,7 @@ This container has two volume mount-points:
 
 The container accepts the following environment variables:
 
+- ```ROOT``` - Root directory for the Reviewboard. 
```

Comment:
How about renaming this to SITE_ROOT?
Also the description in the README is not very self-explaining, can you give an example?

E.g.

```SITE_ROOT``` - The path of the site, relative to the domain. This should start and end with a ```/```. For example, ```/reviews/```. Defaults to ```/```